src/wizard/state.jl

```diff
@@ -35,6 +35,7 @@ necessary.  It also holds all necessary metadata such as input/output streams.
     source_files::Union{Nothing, Vector{String}} = nothing
     source_hashes::Union{Nothing, Vector{String}} = nothing
     dependencies::Union{Nothing, Vector} = nothing
+    compilers::Union{Nothing, Vector} = nothing
```

Comment:
Can you constrain this to be a `Vector{Symbol}`?  I'm extra paranoid about types here since we have to serialize.